src/server/plugins/engine/models/FormModel.ts

```diff
@@ -148,30 +148,10 @@ export class FormModel {
   ) {
     // Build the entire model schema
     // from the individual pages/sections
-    let schema = joi.object().required()
+    const schema = joi.object().required()
 
-    ;[undefined, ...this.sections].forEach((section) => {
-      const sectionPages = relevantPages.filter(
-        (page) => page.section === section
-      )
-
-      if (sectionPages.length) {
-        if (section) {
-          let sectionSchema = joi.object().required()
-
-          sectionPages.forEach((sectionPage) => {
-            sectionSchema = sectionSchema.concat(sectionPage.stateSchema)
-          })
-
-          schema = schema.append({
-            [section.name]: sectionSchema
-          })
-        } else {
-          sectionPages.forEach((sectionPage) => {
-            schema = schema.concat(sectionPage.stateSchema)
-          })
-        }
-      }
+    relevantPages.forEach((page) => {
+      schema.concat(page.stateSchema)
```

Comment:
Can we put the ~`const`~ `let` back above and re-assign the schema every time it changes?

Think we should have code coverage for this (because everything still passed) 😱

```suggestion
      schema = schema.concat(page.stateSchema)
```

Reply:
Ahh!! Good spot, thank you.